DS/DsAssignment4/MainClass.java

```diff
@@ -0,0 +1,53 @@
+import java.util.Scanner;
+
+public class MainClass {
+
+	private void jobProcess() {
+
+		Scanner scanner = new Scanner(System.in);
```

Comment:
ACTION: Why is scanner required for any implementation. Your system (implementation) should be independent of any input. If I want to use your code for my purpose (may be some other scheduling system) then I should not be bounded by the fixed inputs what u had added.